src/Solver/time_stepping/GhostTimeClusterFactory.h

```diff
@@ -26,15 +26,6 @@ struct GhostTimeClusterFactory {
                                               otherGlobalTimeClusterId,
                                               meshStructure);
     }
-    case MPI::DataTransferMode::CopyInCopyOutDevice: {
```

Comment:
I'm wondering here whether it would make sense to keep this copy mode in place - maybe it is useful in the future for prototyping?

Reply:
I don't think so. Device2device copy makes little sense and if one wants to prototype a new feature one can base that on CopyInCopyOutHost